src/domain/app/App.tsx

```diff
@@ -39,4 +40,6 @@ const App = () => {
   );
 };
 
+(window as any).faker = faker;
```

Comment:
Is this used somewhere? I didn't find any usage of window.faker (at least on this PR)

Reply:
Good catch, this was only to test the faker in the browser console (not the best documentation)